sycl/test-e2e/AddressSanitizer/lit.local.cfg

```diff
@@ -5,3 +5,7 @@ config.substitutions.append(
 config.substitutions.append(
     ("%force_device_asan_rt", "env SYCL_PREFER_UR=1 UR_ENABLE_LAYERS=UR_LAYER_ASAN")
 )
+
+# DeviceSanitizer only supports CPU, DG2 and PVC
+if not ('gpu-intel-dg2' in config.available_features or 'gpu-intel-pvc' in config.available_features):
+  config.unsupported = True
```

Comment:
How would it return "supported" for CPU?

Reply:
If the ci machine has dg2 or pvc, the cpu device will also be tested.
I didn't figure out how to enable this test run on cpu even the ci machine doesn't have dg2 and pvc.

Reply:
> If the ci machine has dg2 or pvc, the cpu device will also be tested.

We can't rely on that, you need to come up with something different here.

Reply:
ok, I find a way to add cpu when dg2 or pvc is not available.